database/seeders/MainSeeder.php

```diff
@@ -31,23 +31,26 @@ public function run()
             $mediumData = get_object_vars($x->medium);
             $mediumModel = Medium::firstOrCreate($mediumData);
 
-            //genre
+            //genre and genre_medium
             $genreData = $x->genre;
             foreach ($genreData as $genre){
                 $genreModel = Genre::firstOrCreate(['name' => $genre->name]);
+                $mediumModel->genres()->attach($genreModel->id);
             }
 
-            //production companies
+            //production companies and company_medium
             $productionCompanyData = get_object_vars($x->production_companies);
             $productionCompanyModel = ProductionCompany::firstOrCreate($productionCompanyData);
+            $mediumModel->porduction_companies()->attach($productionCompanyModel->id);
 
-            //countries
+            //countries and country_medium
             $countriesData = $x->countries;
             foreach ($countriesData as $country){
                 $countryModel = Country::firstOrCreate([
                     'code'=> 'abc',
```

Comment:
you should check name and code
'code' => $country->code,

Reply:
Oh well, i had created field in json file and stil left it like this